doc/contributing/writing-tests.md

```diff
@@ -141,6 +141,26 @@ request. Interesting things to notice:
 
 ## General recommendations
 
+### Usage of `node:test`
+
+It is recommended to use `node:test` in tests outside of testing the `node:test`
```

Comment:
Have we agreed that it's *recommended*? I remember us saying it won't hurt, but we won't enforce people to use it.

Reply:
I can change the wording. Can you recommend a new change?

Reply:
https://github.com/nodejs/node/pull/56027#discussion_r1860949930

Reply:
I don't think we should force people one way or the other. But I do think we should explicitly stop using the pattern of creating block scopes to isolate tests. It's frustrating when one of those quasi-tests fails and the rest of the file does not execute.

Reply:
Thanks @RafaelGSS